src/Spell Checker/main.vala

```diff
@@ -0,0 +1,17 @@
+#! /usr/bin/env -S vala workbench.vala --pkg libadwaita-1 --pkg gtksourceview-5 --pkg libspelling-1
+
+public void main () {
+    GtkSource.init ();
+    var buffer = (GtkSource.Buffer) workbench.builder.get_object ("buffer");
+    var text_view = (Gtk.TextView) workbench.builder.get_object ("text_view");
+
+    Spelling.Checker checker = Spelling.Checker.get_default ();
+    checker.language = "en_US";
+    Spelling.TextBufferAdapter adapter = new Spelling.TextBufferAdapter (buffer, checker);
```

Comment:
The constructor makes the type clear, so `var` would be better in this case too.